src/pages/workspaces/workspace/workflows/WorkflowView.js

```diff
@@ -350,14 +352,24 @@ const WorkflowView = _.flow(
     })
   }
 
+  /** Returns the href link for the specified article. */
+  getSupportLink(article) { return `https://support.terra.bio/hc/en-us/articles/${article}` }
+
+  /** Returns true if the increasing memory on retries is enabled */
+  isMemoryIncreasedOnRetries() { return this.state.retryWithMoreMemory }
+
+  /** Returns true if the memory retry factor is outside the expected range */
+  warnMemoryFactor() { return this.state.retryMemoryFactor > 2 }
```

Comment:
nit: With these two functions I prefer directly using the state it is more straightforward to me when I read the code - I don't have to bounce back to the function to see what it is doing.

Reply:
Done.